Update file-system.md (PR#1584 fixes) (#1657)
Samsung/tizen-docs: docs/application/web/guides/data/file-system.md

# File System

> > [!NOTE]  
> > 
> > With the introduction of Tizen version 5.0, the Filesystem API has undergone a major overhaul. Many of the existing methods are deprecated and new methods are introduced. For information on the previous versions of API, see [Deprecated Filesystem Guide](./file-system-old.md). This guide explains about the APIs for Tizen 5.0 and its newly introduced methods.

Reviewer: ```suggestion
> [!NOTE]  
> With the introduction of Tizen version 5.0, the Filesystem API has undergone a major overhaul. Many of the existing methods are deprecated and new methods are introduced. For information on the previous versions of API, see [Deprecated Filesystem Guide](./file-system-old.md). This guide explains about the APIs for Tizen 5.0 and its newly introduced methods.
```

Global.

It is confusable, but the new style note delete the middle empty line.

https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md#alerts-note-tip-important-caution-warning

Author: @mijin85cho, fixed all the note related issues.

      reader.readAsText(fileContentsInBlob);
      ```

   - Close the file when it is not in use:
       ```
       fileHandleRead.close();
       ```

        > [!NOTE]

>         > When you use readString(), readData(), readBlob() or one of their NonBlocking equivalent, FileHandle position is set right where the reading operation ended. To read file again from the beggining, use seek() method on FileHandle object.

Reviewer 2: ```suggestion
        > When you use readString(), readData(), readBlob(), or one of their NonBlocking equivalents, FileHandle position is set right where the reading operation ended. To read the file again from the beginning, use seek() method on FileHandle object.
```

   The method returns a `FileHandle` object, which is a handle to the opened or newly created file.

   > [!NOTE]

>    > openFile() method used with mode 'w' creates new file or opens existing file and erases its contents. Any data contained in a file opened in that mode will be lost.

Reviewer 2: ```suggestion
   > openFile() method used with mode 'w' creates a new file or opens an existing file and erases its contents. Any data contained in a file opened in that mode will be lost.
```
